### PR TITLE
Adjust $service_status for Debian Jessie

### PR DESCRIPTION
**Pull request by raphink**

### CODE REVIEW

**Comment by raphink** [comment]
@mhaskel @hunner this is a simple fix for Debian Jessie.

**Comment by mhaskel** [comment]
thanks @raphink !

**Comment by raphink** [comment]
Thanks. Is there a release planned soon?

**Comment by mhaskel** [comment]
@raphink there isn't one scheduled since we just released this, but I can add this to the list of 'modules we have requests for releases of'.

**Comment by raphink** [comment]
Thanks. In the meanwhile, I will point to the commit here.

File: manifests/params.pp [hunk 1]
```diff
@@ -142,7 +142,12 @@
       $bindir               = pick($bindir, "/usr/lib/postgresql/${version}/bin")
       $datadir              = pick($datadir, "/var/lib/postgresql/${version}/main")
       $confdir              = pick($confdir, "/etc/postgresql/${version}/main")
-      $service_status       = pick($service_status, "/etc/init.d/${service_name} status | /bin/egrep -q 'Running clusters: .+|online'")
+      if versioncmp($::operatingsystemrelease, '8.0') >= 0 {
+        # Jessie uses systemd
+        $service_status = pick($service_status, "/usr/sbin/service ${service_name}@*-main status")
+      } else {
+        $service_status = pick($service_status, "/etc/init.d/${service_name} status | /bin/egrep -q 'Running clusters: .+|online'")
```
Comment (igalic) [inline manifests/params.pp]: how do we create the cluster on Debian?
it seems that ubuntu (14.04+) will need that too…

Comment (mhaskel) [inline manifests/params.pp]: this definitely failed in acceptance tests on ubuntu 14.04. thus #588

Comment (raphink) [inline manifests/params.pp]: Ah right, sorry for that.

Comment (mhaskel) [inline manifests/params.pp]: no worries. I didn't catch it until ci did :)

**Comment by frenkel** [comment]
I just upgraded to Debian Jessie, and using master I always get:

    Notice: /Stage[main]/Postgresql::Server::Service/Service[postgresqld]/ensure: ensure changed 'stopped' to 'running'

Was this pull-request supposed to fix that?

**Comment by mhaskel** [comment]
@frenkel hrm, yeah. That doesn't seem quite right. Can you submit a ticket at https://tickets.puppetlabs.com/browse/MODULES so we can track this?

**Comment by frenkel** [comment]
@mhaskel Thanks, didn't know about that site. I've submitted an issue (https://tickets.puppetlabs.com/browse/MODULES-2123)

**Pull request merged**